src/coreComponents/unitTests/meshTests/supportedElements.vtk

```diff
@@ -0,0 +1,571 @@
+# vtk DataFile Version 4.2
+vtk output
+ASCII
+DATASET UNSTRUCTURED_GRID
+
+POINTS 135 float
```

Comment:
```suggestion
DATASET UNSTRUCTURED_GRID

FIELD FieldData 1
attributeName 2 11 string
0 
Tetrahedron
1 
Pyramid
2 
Wedge
3 
Hexahedron
4 
PentagonalPrism
5 
HexagonalPrism
6 
HeptagonalPrism
7 
OctagonalPrism
8 
NonagonalPrism
9 
DecagonalPrism
10 
HendecagonalPrism

POINTS 135 float
```

Just a suggestion to start using the  `FIELD` dataset format to provide labels for element region markers.